### ISSUE TITLE
3rd party wallet connection [Must have]

### ISSUE THREAD

**Issue by EstherCentrifuge** [centrifuge/apps#841]
@onnovisser I think the options are https://github.com/Koniverse/SubConnect/tree/master/src/lib/wallets or https://github.com/TalismanSociety/talisman-connect/tree/master/libs/wallets. Would be great if you could check out both and compare. Apparently the former is based on the latter but has some changes, I’m not really sure of the details

**Comment by sophialittlejohn** [comment]
Feedback on [Slack](https://centrifugehq.slack.com/archives/C0376B9ES5C/p1655133715329869?thread_ts=1655108732.180869&cid=C0376B9ES5C)

> so something random happened to me just now.
I had an open transaction and i didn't sign nor cancelled.
I refreshed the page and my wallet got disconnected. I selected polkadot JS and it went to the extension page of the addon.
when i cancelled the transaction i could reconnect

> also when you reresh the page it gets the top user in the wallet, it doesn't stick with the one you picked before
